kits/survey/contracts/SurveyKit.sol

```diff
@@ -43,16 +42,17 @@ contract SurveyKit is APMNamehash, KitBase {
         acl.createPermission(this, dao, dao.APP_MANAGER_ROLE(), this);
 
         Survey survey = Survey(dao.newAppInstance(SURVEY_APP_ID, latestVersionAppBase(SURVEY_APP_ID)));
-
-        // Set escapeHatch address as the default vault, in case a token rescue is required
-        dao.setApp(dao.APP_BASES_NAMESPACE(), dao.DEFAULT_VAULT_APP_ID(), escapeHatch);
-
         survey.initialize(signalingToken, participation, duration);
 
+        // Install a vault to let it be the escape hatch
+        Vault vault = Vault(dao.newAppInstance(KERNEL_DEFAULT_VAULT_APP_ID, latestVersionAppBase(KERNEL_DEFAULT_VAULT_APP_ID)));
+        vault.initialize();
+
         // Set survey manager as the entity that can create votes and change participation
         // surveyManager can then give this permission to other entities
         acl.createPermission(surveyManager, survey, survey.CREATE_SURVEYS_ROLE(), surveyManager);
         acl.createPermission(surveyManager, survey, survey.MODIFY_PARTICIPATION_ROLE(), surveyManager);
+        acl.createPermission(surveyManager, vault, vault.TRANSFER_ROLE(), surveyManager);
         acl.grantPermission(surveyManager, dao, dao.APP_MANAGER_ROLE());
         acl.setPermissionManager(surveyManager, dao, dao.APP_MANAGER_ROLE());
 
```

Comment:
For some reason, `cleanupDAOPermissions()` always fails for me... it keeps complaining about the role not having an manager set (`ACL_AUTH_NO_MANAGER`).

cc @izqui

Reply:
Yes, there's a function [`cleanupPermission`](https://github.com/aragon/dao-kits/blob/master/kits/base/contracts/KitBase.sol#L37) for cleaning permissions individually, but it's better to remove that line and rely on the other one. Actually the line before (L56) can be removed, as the permission is also set in `cleanupDAOPermissions`.